ohsome_quality_api/topics/presets.yaml

```diff
@@ -547,7 +547,7 @@ mapaction-rivers-length:
 
 ### IDEALVGI PROJECT ###
 
-lulc:
+land-cover:
   name: Land Use and Land Cover
   description: >-
     Features related to land use and land cover.
```

Comment:
This description is probably not right anymore. Lets shorten it to the first sentence.